lib/coffee_app.dart

```diff
@@ -1,16 +1,24 @@
 import 'package:coffee_app/common/theme.dart';
+import 'package:coffee_app/features/screens/home_screen.dart';
 import 'package:coffee_app/router/router.dart';
 import 'package:flutter/material.dart';
 
+import 'features/rewards/screen/rewards_screen.dart';
```

Comment:
No relative imports, every import should have 'package'-like structure, i.e. `package:coffee_app/features/screens/home_screen.dart`